protocol/app/prepare/prepare_proposal.go

```diff
@@ -75,8 +77,9 @@ func PrepareProposalHandler(
 		}
 
 		// Grab the injected VEs from the previous block.
-		// If VEs are not enabled, no tx will have been injected.
 		var extCommitBzTx []byte
+		// Sanity check to ensure that there is at least 1 tx. This should never return false unless
+		// before VE are enabled, there are no tx in the block.
```

Comment:
I am confused around "there are no tx in the block". Shouldn't regular user submitted txs be part of req.Txs?

Reply:
what I meant is that if there are no user submitted txs to be a part of req.Txs, then the block would be empty right?

Reply:
I see. I think we should file a ticket and add TODOs to remove any "do this before VE is enabled" logic once v5.0 ships.

What do you think?

Reply:
I can file a ticket to just clean it up, I think adding the TODOs at this point would be more work

Reply:
https://linear.app/dydx/issue/SKI-36/cleanup-all-code-that-only-triggers-if-ve-is-not-enabled